easybuild/easyblocks/h/hdf5.py

```diff
@@ -84,6 +84,9 @@ def configure_step(self):
             mpich_mpi_families = [toolchain.INTELMPI, toolchain.MPICH, toolchain.MPICH2, toolchain.MVAPICH2]
             if self.toolchain.mpi_family() in mpich_mpi_families:
                 self.cfg.update('buildopts', 'CXXFLAGS="$CXXFLAGS -DMPICH_IGNORE_CXX_SEEK"')
+            # Skip MPI cxx extensions to avoid hard dependency
+            if self.toolchain.mpi_family() in toolchain.OPENMPI:
```

Comment:
Hmm, should be `==` rather than `in`?

Reply:
Oh... How did this work then? Double-checking

Reply:
It did work though: `== 2020-01-14 11:16:31,794 run.py:534 INFO cmd " make -j 12  CXXFLAGS="$CXXFLAGS -DOMPI_SKIP_MPICXX"  FC="mpifort" " exited with exit code 0 and output:
`

Reply:
Fixed in https://github.com/easybuilders/easybuild-easyblocks/pull/1919

Reply:
It worked because `toolchain.OPENMPI` is a string value (but you shouldn't rely on that, in theory it could change).